doc/classes/Vector2.xml

```diff
@@ -311,6 +312,7 @@
 			<param index="0" name="n" type="Vector2" />
 			<description>
 				Returns the vector reflected (i.e. mirrored, or symmetric) over a line defined by the given direction vector [param n].
+				[b]Note:[/b] [method reflect] performs [i]mathematical[/i] reflection, as opposed to [method bounce] which performs [i]physical[/i] reflection. Double-check which method you actually need to use, as other engines may use these terms differently.
```

Comment:
> performs [i]mathematical[/i] reflection

I'm not sure that this is properly called "mathematical" reflection. From what I can see, "n" is always the normal vector (i.e. pointing perpendicular from the surface). While in this case "n" is not the normal, but the plane itself.

Reply:
A little more insight, the wikipedia page for "reflection (mathematics)" gives a definition for reflection the same as what we use for reflection, but calls the parameter "line" instead of "normal". (https://en.wikipedia.org/wiki/Reflection_(mathematics))

Bounce on the other hand, is a mathematical reflection over a normal. It is also equivalent to ``-vec.reflect()``

Instead of calling them "mathematical reflection" and "physical reflection" I would highlight that you are reflecting over a normal vector perpendicular to the plane, or a vector describing the plane itself. So you are either "bouncing off a surface with normal n" or you are "reflecting across the surface n"

Reply:
> So you are either "bouncing off a surface with normal n"

Isn't that what `bounce()` does?